test/test_functions.jl

```diff
@@ -51,7 +51,7 @@ function jac_eval(rs::ReactionSystem, u, p, t; combinatoric_ratelaws = true, spa
     prob = ODEProblem(rs, u, 0.0, p; jac = true, combinatoric_ratelaws, sparse)
     J = sparse ? deepcopy(prob.f.jac_prototype) : zeros(length(u), length(u))
     prob.f.jac(J, prob.u0, prob.p, t)
-    @test J == prob.f.jac(prob.u0, prob.p, t)
+    @test J ≈ prob.f.jac(prob.u0, prob.p, t)
```

Comment:
Use a higher tolerance. Seems like they should agree to basically floating point precision (so maybe check agreement to one or two lower orders)?

Reply:
will do, think there might be some other cases where his happens as well. When I do I will add like `1e-14` to them and we should be good.

Reply:
Does this one need a tolerance update to be a bit more strict (like the others)?

Reply:
agree, this one should have as well